docroot/profiles/sdss/sdss_profile/themes/sdss_subtheme/src/scss/components/row-backgrounds/_component-overrides.scss

```diff
@@ -172,5 +205,50 @@
     }
   }
 
+  .paragraph--type--stanford-entity {
+    .ds-entity--stanford-publication {
+      .su-card__contents {
+        .su-publication-topics {
+          a {
+            color: $su-color-white;
+
+            &:hover,
+            &:focus,
+            &:active {
+              color: $sdss-color-white;
+              text-decoration: underline;
+            }
+          }
+        }
+
+        span {
+          color: $su-color-white;
+        }
+
+        > h2 {
+          a {
+            color: $sdss-color-white;
+
+            &::after {
+              background-color: $sdss-color-white;
+              width: 2.5rem;
+              height: 2.5rem;
+            }
+
+            &:hover,
+            &:focus,
+            &:active {
+              color: $sdss-color-white;
+              text-decoration: underline;
+
+              &::after {
+                background-color: $sdss-color-white;
+              }
+            }
```

Comment:
Shouldn't this be `$sdss-color-green-bright;` on hover, to match the behavior you are adding for the shared tags publications card?

Reply:
That was a great find @jdwjdwjdw! I removed a bunch of code and just added white to the a so when images are turned off you don't get the red color on green.